cirq-google/cirq_google/engine/engine_processor.py

```diff
@@ -14,13 +14,14 @@
 
 import datetime
 
+import logging
```

Comment:
Do you know if there's a policy on logging in Cirq? I only see it used for tests in Google-managed code (cirq-core and cirq-google). I do see some examples of print statements coupled with a verbosity flag, e.g.: https://github.com/quantumlib/Cirq/blob/4afe4aebb36e56f0f7dac689c671454268eac0c3/cirq-google/cirq_google/engine/engine_client.py#L151